app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/Address.cs

```diff
@@ -29,6 +29,6 @@ public class Address : PersonalInfo
         public string ZipPostalCode { get; set; }
 
         [Description("The owner of the address e.g. Applicant, Person Sought, e.t.c")]
-        public OwnerType Owner { get; set; }
+        public OwnerType Owner { get; set; } = OwnerType.NotApplicable;
```

Comment:
We shouldn't put this in the contract. It should be in the mapping or separate definition